src/Nethermind/Nethermind.TxPool/TxPoolConfig.cs

```diff
@@ -7,6 +7,12 @@ public class TxPoolConfig : ITxPoolConfig
     {
         public int PeerNotificationThreshold { get; set; } = 5;
         public int Size { get; set; } = 2048;
+        public bool PersistentBlobStorageEnabled { get; set; } = false;
+        public int PersistentBlobStorageSize { get; set; } = 16 * 1024; // theoretical max - 12,5GB (128KB * 6 * 16384); for one-blob txs - 2GB (128KB * 1 * 16384);
```

Comment:
I don't like this idea. `16.KiB()` means 16 kilobytes, and here `PersistentBlobStorageSize` means number of transactions.
I understand that `KiB` will work here, but it will be misleading as well - it is not memory size, it is numerical size of collection.
If you insist, I can change it. But IMO it is not a good change.
Or maybe we should add more universal `Ki()` to size extensions, which will be the same as `KiB`, but not suggesting connection with bytes? But I never see `Ki` in different context than kilobytes so just `KiB`, so it will be strange as well 😅